msix.catalog.wad.test/TestSettings1.testsettings

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why did you add a testSettings?

TestSettings will soon be deprecated for automated unit and functional testing scenarios. It is recommended that you use RunSettings. To learn more, see http://aka.ms/runsettings